publication-rest/src/main/java/no/unit/nva/publication/fetch/FetchPublicationHandler.java

```diff
@@ -127,7 +123,10 @@ private String produceRemovedPublicationResponse(Publication publication, Reques
         if (nonNull(publication.getDuplicateOf()) && shouldRedirect(requestInfo)) {
             return produceRedirect(publication.getDuplicateOf());
         } else {
-            var allowedOperations = getAllowedOperations(requestInfo, publication);
+            var publicationStrategy = getPublicationPermissionStrategy(requestInfo, publication);
+
+            Set<PublicationOperation> allowedOperations =
```

Comment:
This can probably be `.map(PublicationStrategy::getAllowedActions).orElse(emptySet())`

Reply:
nice!